templates/embed_404.html

```diff
@@ -1,12 +1,13 @@
 {%  extends 'base.html' %}
 {% load render_bundle from webpack_loader %}
 {% load account %}
+{% load i18n_icu %}
 
 {% block html_tag_attrs %}
     class="embed_container"
 {% endblock %}
 
 {% block body %}
-    <h1>This workflow is not available :(</h1>
+    <h1>{% trans_html "py.templates.embed_404.notAvailable.header" default="This workflow is not available :(" %}(</h1>
```

Comment:
Please nix the `(`

Reply:
If we nix the `(`, we must also nix the `:`, since this is an emoticon